psql2mysql/__init__.py

```diff
@@ -167,13 +173,18 @@ def disable_constraints(self):
             "SET SESSION foreign_key_checks='OFF'"
         )
 
+    @profile
     def writeTableRows(self, table, rows):
-        # FIXME: Allow to process this in batches instead one possibly
-        # huge transcation?
-        self.connection.execute(
-            table.insert(),
-            rows.fetchall()
-        )
+        if self.chunked and self.chunk_size > 0:
```

Comment:
You already check it's a positive integer in setup(), so I think just if self.chunked is fine here.

Reply:
ahh yes. I wanted 0 to be turned off (migrate the whole table) but in debugging (ie sending none to fetchmany) actually set it to 1, so not what I wanted. Added to > 0 to force it to the else, meant to go back and clean it up.